Cairo Desktop/Cairo Desktop/Services/DesktopManager.cs

```diff
@@ -197,9 +200,12 @@ private void TeardownDesktop()
             DestroyDesktopBrowser();
             DestroyDesktopWindow();
             DestroyShellWindow();
+            ShowWindowsDesktopIcons();
+        }
 
-            // show the windows desktop
-            ShellHelper.ToggleDesktopIcons(true);
+        private static void ShowWindowsDesktopIcons()
+        {
```

Comment:
I think it would be cleaner to have a single toggle function that is passed a boolean for show/hide, rather than two functions that differ by one true/false.